providers/bnxt_re/verbs.c

```diff
@@ -102,6 +102,20 @@ static int bnxt_re_map_db_page(struct ibv_context *ibvctx,
 	return 0;
 }
 
+int bnxt_re_notify_drv(struct ibv_context *ibvctx)
+{
+	DECLARE_COMMAND_BUFFER(cmd,
+			       BNXT_RE_OBJECT_NOTIFY_DRV,
+			       BNXT_RE_METHOD_NOTIFY_DRV,
+			       0);
+	int ret;
+
+	ret = execute_ioctl(ibvctx, cmd);
```

Comment:
return execute_ioctl(); directly without need in ret.

Reply:
> return execute_ioctl(); directly without need in ret.

will do. thanks